civihr_employee_portal/js/ta-documents-app.js

```diff
@@ -0,0 +1,68 @@
+/* globals angular */
+
+(function (angular) {
+  angular.module('taDocuments', ['civitasks.appDocuments']).controller('ModalController', ['$scope', '$rootScope', '$window', '$rootElement', '$log', '$uibModal', 'DocumentService', 'FileService', 'config',
+    function ($scope, $rootScope, $window, $rootElement, $log, $modal, DocumentService, FileService, config) {
+      var vm = {};
+
+      /**
+       * Gets Document for the given document id and
+       * opens Modal with retrived document data
+       *
+       * @param {integer} id
+       * @param {string} role
+       */
+      vm.modalDocument = function (id, role) {
+        DocumentService.get({ id: id })
+          .then(function (data) {
+             if (!data) {
+               throw new Error('Requested Document is not available');
+             }
+
+             openModalDocument(data[0], role);
+          })
+          .catch(function (reason) {
+            CRM.alert(reason, 'Error', 'error');
+          });
+      }
+
+      ;(function init ($window) {
```

Comment:
why is this semicolon here? the problem is that the semicolon was missing from here
```js
vm.modalDocument = function (id, role) {

}; // <-- missing semicolon